cli/deno_dir.rs

```diff
@@ -61,6 +108,36 @@ impl DownloadCache {
   }
 }
 
+// TODO: this is a structure that is supposed to be moved to //core/cache.rs
+#[derive(Clone)]
```

Comment:
I know naming is hard, but you kinda had me with 'DiskCache'. Typically it is used for on-disk files cached in RAM.
Maybe something like "LocalCache" or "FetchCache" would be better.

Reply:
> I know naming is hard, but you kinda had me with 'DiskCache'. Typically it is used for on-disk files cached in RAM.

Hmmm, good point `LocalCache` seems right to me. 

We'd still need disk files cached in RAM to avoid multiple FS trips, but I was supposed to name that `InProcessCache` or `SourceFileCache`. 

Happy to hear your opinions.

Reply:
@piscisaureus are you adamant on renaming `DiskCache` to `LocalCache`?

Reply:
@bartlomieju It's not super important to me.